includes/blocks/course-theme/class-the-content.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * File containing the Sensei\Blocks\Course_Theme\The_Content class.
+ *
+ * @package sensei
+ * @since 4.0.0
+ */
+
+namespace Sensei\Blocks\Course_Theme;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use \Sensei_Blocks;
+
+/**
+ * Block to render the content for the current lesson or quiz page.
+ */
+class The_Content {
+
+	/**
+	 * Content constructor.
+	 */
+	public function __construct() {
+		Sensei_Blocks::register_sensei_block(
+			'sensei-lms/the-content',
```

Comment:
Not sure what's the best name for this. Would `course-content` be confusing?

Reply:
I like `course-content` considering this will be `wp:post-content` in the future. It seems a good pair to me. :)

Reply:
Renamed the new blocks in 70a0d18